<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDiskMark</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="loopsCount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>42</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_FSPoints">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="readBar_SEQ1M_Q8T1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;p align=&quot;center&quot;&gt;Read [MiB/s]&lt;/p&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEQ1M_Q8T1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SEQ1M
Q8T1</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="writeBar_SEQ1M_Q8T1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;p align=&quot;center&quot;&gt;Write [MiB/s]&lt;/p&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>79</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MB/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GB/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IOPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Î¼s</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="readBar_SEQ1M_Q1T1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="writeBar_SEQ1M_Q1T1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SEQ1M_Q1T1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SEQ1M
Q1T1</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="writeBar_RND4K_Q32T16">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="readBar_RND4K_Q32T16">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RND4K_Q32T16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RND4K
Q32T16</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="readBar_RND4K_Q1T1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RND4K_Q1T1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RND4K
Q1T1</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="writeBar_RND4K_Q1T1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuInterval_Time">
     <property name="title">
      <string>Interval Time</string>
     </property>
     <addaction name="action0_sec"/>
     <addaction name="action1_sec"/>
     <addaction name="action3_sec"/>
     <addaction name="action5_sec"/>
     <addaction name="action10_sec"/>
     <addaction name="action30_sec"/>
     <addaction name="action1_min"/>
     <addaction name="action3_min"/>
     <addaction name="action5_min"/>
     <addaction name="action10_min"/>
    </widget>
    <addaction name="menuInterval_Time"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action0_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 sec</string>
   </property>
  </action>
  <action name="action1_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 sec</string>
   </property>
  </action>
  <action name="action3_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 sec</string>
   </property>
  </action>
  <action name="action5_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 sec</string>
   </property>
  </action>
  <action name="action10_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 sec</string>
   </property>
  </action>
  <action name="action30_sec">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30 sec</string>
   </property>
  </action>
  <action name="action1_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </action>
  <action name="action3_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 min</string>
   </property>
  </action>
  <action name="action5_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 min</string>
   </property>
  </action>
  <action name="action10_min">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 min</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About KDiskMark</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
